src/BlockID.cpp

```diff
@@ -222,6 +236,11 @@ bool StringToItem(const AString & a_ItemTypeString, cItem & a_Item)
 	{
 		ItemName = ItemName.substr(10);
 	}
+
+	if (gsBlockIDMap.m_bHasRunInit != true)
```

Comment:
This is an artefact of some coding standards I normally use. 

For the sake of readability, I usually do the above, and let the compiler optimise it out. Is the rest of the codebase using the short form?